web/src/app/modules/views/side/modules/history-view/components/history-view.component.html

```diff
@@ -0,0 +1,18 @@
+<div *ngIf="modelHistoryEntries" class="card">
+  <h5 class="card-header">{{'Changes' | translate}}</h5>
+  <ul class="list-group list-group-flush">
+    <li *ngFor="let entry of modelHistoryEntries" class="list-group-item">
+      <span class="data-item-title">{{entry.user}}</span>&nbsp;<small class="text-muted">{{entry.date.toLocaleString()}}</small>
+        <ul>
+            <li *ngFor="let change of entry.changes" class="list-unstyled">
+              <div *ngIf="change.isDelete">
+                {{'ElementDeleted' | translate}}
+              </div>
+              <div *ngIf="!(change.isCreate && change.isDelete)">
+                {{'Changed' | translate}} <span class="data-item-title">{{change.feature}}</span>&nbsp;<span class="fa fa-arrow-right" aria-hidden="true">&nbsp;{{change.newValue}}</span>
```

Comment:
The span with the icon leads to ill-rendered font in the text behind the icon. Place the icon in a span- or i-tag and close it immediately.